### DISCUSSION TITLE
[Feature] Pubmed

### DISCUSSION THREAD

**Discussion by 157655** [windingwind/zotero-actions-tags#155] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS: Mac OS vetura 13.6
- Zotero Version: 7
- Plugin Version: newest

### Describe the feature request

When I import literature in pubmed, zotero connector always annoyingly adds an attachment named "pubmed entry". Can I use your plugin to automatically remove this attachment? Or is there a more fundamental way to prevent adding this attachment?

### Describe the solution you'd like

make world better

### Anything else?

。

**Comment by windingwind** [comment]
> Can I use your plugin to automatically remove this attachment?

Yes.

Maybe someone who is familiar with Pubmed can help. If you can provide some screenshots of the attachment `Pubmed entry`, I can have a try.

**Reply by yuzuan** [reply]
![WeChatad88829dbe52c3cabcde8e8e6cf74914](https://github.com/windingwind/zotero-actions-tags/assets/54974291/55d1681d-5621-4c56-87a9-1740ed4a89a1)

**Reply by windingwind** [reply, marked as answer]
Seem like this is a snapshot, which is a Zotero feature. You can disable it here:

<img width="800" alt="image" src="https://github.com/windingwind/zotero-actions-tags/assets/33902321/280f5b4c-08fc-4d4b-8176-625732ae0b6c">
